ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
s/`to help us improve`/`to file an error or unexpected behaviour`? Improvement is more like an enhancement suggestion or feature request.